server/plugin.ts

```diff
@@ -15,6 +15,7 @@
 
 import { first } from 'rxjs/operators';
 import { Observable } from 'rxjs';
+import _ from 'lodash';
```

Comment:
We can remove this import if it is not being used

Reply:
done